org.eclipse.xtext.xtext.ui.examples/projects/arithmetics/org.eclipse.xtext.example.arithmetics/src/org/eclipse/xtext/example/arithmetics/GenerateArithmetics.mwe2

```diff
@@ -62,6 +62,9 @@ Workflow {
 				generateStub = true
 				generateXtendStub = false
 			}
+			junitSupport = {
+				generateXtendStub = false
```

Comment:
Should this be covered by preferXtendStubs above?

Reply:
no cause backwards compatibility :(